docs/releases.md

```diff
@@ -83,6 +83,14 @@ Patch releases will usually follow a compressed release cycle and should take 2-
 
 Some patch releases, especially ones fixing one or more complex bugs, may undergo the full release process.
 
+## Security Fix Policy
+
+Any release may contain security fixes. Unless the fix addresses a bug being exploited in the wild, the fix will _not_ be called out in the release notes. Please make sure to update ASAP.
```

Comment:
This should at least be encoded as a minor version update, I don't know how realistic it is to offload the responsibility of monitoring and updating to any release.

Reply:
A mailing list may be a better approach here for notification than relying on versioning. We could then link the opt in list in the security readme. I don't want to block this on that decision though.

Reply:
Patch releases may also include super-critical security releases, although I guess those will usually be announced (or really well hidden but that's difficult).